ui/widgets/datepicker.js

```diff
@@ -79,30 +81,33 @@ var widget = $.widget( "ui.datepicker", {
 	_getCreateOptions: function() {
 		var max = this.element.attr( "max" ),
 			min = this.element.attr( "min" ),
+			parser = new Globalize( "en" ).dateParser( { raw: "yyyy-MM-dd" } ),
```

Comment:
Not directly related to this PR, but does Globalize always contain the `en` data or do we need to tell users to always include this to handle the attribute parsing?